securedrop/i18n.py

```diff
@@ -39,7 +39,7 @@ def setup_app(config, app):
     global LOCALES
     global babel
 
-    translation_dirs = getattr(config, 'TRANSLATION_DIRS', None)
+    translation_dirs = config.TRANSLATION_DIRS
```

Comment:
@heartsucker I can not explain how happy I am to see these `*attr` calls being removed :heart: .